src/shared/colors/TableRowColors.js

```diff
@@ -0,0 +1,15 @@
+import { cls } from '../cls.js';
+
+export const TableRowColors = (colorsProp = {}, dark) => {
+  return {
+    bgIos: cls(
+      'hover:bg-black hover:bg-opacity-5',
```

Comment:
`hover:bg-black/5` - does the same in much shorter way, try using this everywhere